docs/tutorial.md

```diff
@@ -96,7 +96,7 @@ Let's also prevent players from being able to overwrite cells.
 
 ```js
 function IsVictory(cells) {
-  // Return true if cells is in a winning configuration.
```

Comment:
This is actually referring to the object `cells`, so the singular *is* is correct here. Feel free to wrap cells in quotes if that makes it clearer.